recipes/recipes_emscripten/qutip/test_qutip.py

```diff
@@ -0,0 +1,24 @@
+
+def test_basic_function():
+	from qutip import *
+	print(Qobj())
+
+def test_more_functions():
+	from qutip import *
+	import numpy as np
+
+	coherent(5,0.5-0.5j)
+	q = destroy(4)
+	assert(q.dims == [[4], [4]])
+	assert(q.shape == (4, 4))
+	assert(q.isherm == False)
+	assert(q.type == 'oper')
+
+	x = sigmax()
+
+	a = q + 5
```

Comment:
This will raise an error because `+` is matrix addition for Qobj instances and `q` and `5` have different shapes.

Reply:
I was trying to replicate this section: https://qutip.org/docs/latest/guide/guide-basics.html#qobj-math

:)